helm/hpcc/values.yaml

```diff
@@ -437,9 +437,16 @@ esp:
     ## Annotations can be specified on a service - for example to specify provider-specific information such as service.beta.kubernetes.io/azure-load-balancer-internal-subnet
     #annotations:
     #  service.beta.kubernetes.io/azure-load-balancer-internal-subnet: "mysubnet"
+    #  hpcc.eclwatch.io/enabled: "true"
+    #  hpcc.eclwatch.io/description: "some description"
     ## You can also specify labels on a service
     #labels:
     #  mylabel: "3"
+    ## Links specify the web links for a service. The web links may be shown on ECLWatch.
```

Comment:
These links are static (likely external to k8s cluster) and should only be acknowledged in service:esp/application:eclwatch/links

Reply:
Will add comment per offline discussion.